colorable_windows.go

```diff
@@ -1,3 +1,5 @@
+// +build !appengine
```

Comment:
please fix like my said and copy colorable_others.go to copy_appengine.go like this. https://github.com/mattn/go-isatty/blob/master/isatty_appengine.go